sqlite_query.c

```diff
@@ -690,3 +723,34 @@ static char * sqlite_text_value_to_pg_db_encoding(sqlite3_stmt * stmt, int stmt_
 		/* There is no UTF16 in PostgreSQL for fast sqlite3_column_text16, hence always convert */
 		return (char *) pg_do_encoding_conversion((unsigned char *) utf8_text_value, strlen(utf8_text_value), PG_UTF8, pg_database_encoding);
 }
+
+/*
+ * Converts int64 frrom SQLite to PostgreSQL string from 0 and 1 only
+ */
+char *int2binstr(sqlite3_int64 num, char *s, size_t len)
```

Comment:
It better to use `long2binstr` or `int64tobinstr` to make consistent with function description.

Reply:
Thank for updating. Please help me resolve this thread.